cmd/swaprecover/main.go

```diff
@@ -77,8 +105,9 @@ var (
 				Usage: "Ethereum chain ID; eg. mainnet=1, goerli=5, ganache=1337",
 			},
 			&cli.UintFlag{
-				Name:  flagGasPrice,
-				Usage: "Ethereum gas price to use for transactions (in gwei). If not set, the gas price is set via oracle.",
+				Name:   flagGasPrice,
+				Usage:  "Ethereum gas price to use for transactions (in gwei). If not set, the gas price is set via oracle.",
+				Hidden: true, // Not well tested, hiding from end users for now
```

Comment:
@noot Just wanted to make sure that you saw that I hid the gas price flag. I didn't want to set end-user expectations that the flag is well tested and works.

Reply:
good to know. for deploying on L2s we'll need to document this (or add hard coded values per chain) but this is fine for now